package.json

```diff
@@ -24,7 +24,7 @@
     "@digitalbazaar/ed25519-signature-2018": "^2.1.0",
     "@digitalbazaar/ed25519-signature-2020": "^3.0.0",
     "@digitalbazaar/vc": "^2.1.0",
-    "@digitalbazaar/vc-status-list": "^2.1.0",
+    "@digitalbazaar/vc-status-list": "digitalbazaar/vc-status-list#add-positive-test",
```

Comment:
Once this is updated I'll approve and merge this. We'll have to figure out what to do wrt. existing deployments elsewhere regarding whether this should be pushed out or not.

Reply:
Updated [b7ad589](https://github.com/digitalbazaar/bedrock-vc-verifier/pull/29/commits/b7ad58907ebc313c39fd7585c075c573265354b7)